vowpalwabbit/core/src/parse_args.cc

```diff
@@ -523,6 +523,25 @@ const char* are_features_compatible(VW::workspace& vw1, VW::workspace& vw2)
   return nullptr;
 }
 
+namespace details
+{
+std::tuple<std::string, std::string> extract_ignored_feature(VW::string_view namespace_feature)
+{
+  std::tuple<std::string, std::string> extracted_ns_and_feature;
+  std::string feature_delimiter = "|";
+  auto feature_delimiter_index = namespace_feature.find(feature_delimiter);
+  if (feature_delimiter_index != VW::string_view::npos)
+  {
+    auto ns = namespace_feature.substr(0, feature_delimiter_index);
+    // check for default namespace
+    if (ns.empty()) { ns = " "; }
```

Comment:
I didn't know that but looks like we have counterintuitive feature collision in that kind of example:
... "feature": 1, " ": {"feature": 2} ...
@jackgerrits , is it something known?

Reply:
maybe represent root namespace using some unprintable char internally?

Reply:
let me create bug on that and we can discuss (or decline) there